include/cvc5/cvc5_types.h

```diff
@@ -253,6 +253,25 @@ enum ProofComponent
 /** Writes a proof component identifier to a stream. */
 std::ostream& operator<<(std::ostream& out, ProofComponent pc) CVC5_EXPORT;
 
+/**
+ * Proof format used for proof printing.
+ */
+enum ProofFormat
```

Comment:
Just a reminder that this will have to change when #9915 is merged back. The enum values don't have to be prefixed with `PROOF_FORMAT` then either.